client/src/app/site/pages/meetings/modules/participant-search-selector/components/participant-search-selector/participant-search-selector.component.ts

```diff
@@ -0,0 +1,123 @@
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
+import { FormBuilder, FormGroup } from '@angular/forms';
+import { BehaviorSubject } from 'rxjs';
+import { Permission } from 'src/app/domain/definitions/permission';
+import { BaseUiComponent } from 'src/app/ui/base/base-ui-component';
+
+import { ParticipantControllerService } from '../../../../pages/participants/services/common/participant-controller.service';
+import { ViewUser } from '../../../../view-models/view-user';
+import { UserSelectionData } from '../..';
+
+@Component({
+    selector: `os-participant-search-selector`,
+    templateUrl: `./participant-search-selector.component.html`,
+    styleUrls: [`./participant-search-selector.component.scss`]
+})
+export class ParticipantSearchSelectorComponent extends BaseUiComponent implements OnInit {
+    /**
+     * To check permissions in the template
+     */
+    public readonly permission = Permission;
+
+    private _filteredUsersSubject = new BehaviorSubject<ViewUser[]>([]);
+
+    /**
+     * Subject that holds all users, that should not be selectable.
+     * Needs to be ubdated by parent component.
+     */
+    @Input()
+    public nonSelectableUsersSubject!: BehaviorSubject<number[]>;
```

Comment:
Please avoid expecting a `BehaviorSubject` as input. It has to be either an `Observable` or you write it as setter, so you know when it changes.

Like this:
```suggestion
    public set nonSelectableUsers(users: Identifiable[]) {
```